packages/ui/src/DateTimeField.tsx

```diff
@@ -108,9 +108,9 @@ export const DateTimeField = ({
         const month = cleanedInput.slice(0, 2);
         const day = cleanedInput.slice(2, 4);
         const year = cleanedInput.slice(4, 8);
-        parsedDate = DateTime.fromFormat(`${month}${day}${year}`, "MMddyyyy", {
-          zone: timezone,
-        });
+        parsedDate = DateTime.fromFormat(`${month}${day}${year}`, "MMddyyyy", {zone: timezone})
```

Comment:
![Screenshot 2024-07-17 at 10 20 58 AM](https://github.com/user-attachments/assets/b21a82d1-66f0-4227-9690-1d2bdca17cd4)

this is more of a general note than specific to this PR, but how do we want to handle this? Especially thinking about on mobile, it doesn't seem uncommon for this text to be wider than the area of the field.